activemq/CHANGELOG.md

```diff
@@ -1,8 +1,10 @@
 # CHANGELOG - activemq
 
-0.1.0/ Unreleased
+1.0.0/ Unreleased
```

Comment:
I feel that if anyone ends in the changelog for whatever reason they should know that the actual changelog for the check is not limited to that. Maybe we can parking lot tomorrow?